pinot-core/src/main/java/org/apache/pinot/core/query/request/context/utils/QueryContextUtils.java

```diff
@@ -75,6 +78,14 @@ public static boolean isAggregationQuery(QueryContext query) {
         return true;
       }
     }
+    if(query.getOrderByExpressions() != null) {
```

Comment:
Since we are adding to aggregationInfo already, we should probably check for aggregation info

Reply:
there is no concept of aggregations in the QueryContext

Reply:
My bad, I forgot we moved to querycontext already.